### PR TITLE
[api] make GetBlockMetas() compatible with Babel

### PR DESCRIPTION
**Pull request by Liuhaai**

### CODE REVIEW

**Comment by dustinxie** [comment]
add test in both grcpserver_test.go and web3server_test.go to cover this case

File: api/coreservice.go [hunk 1]
```diff
@@ -1038,11 +1038,11 @@ func (core *coreService) BlockMetas(start uint64, count uint64) ([]*iotextypes.B
 
 	var (
 		tipHeight = core.bc.TipHeight()
-		res       []*iotextypes.BlockMeta
+		res       = make([]*iotextypes.BlockMeta, 0)
 	)
 	if start > tipHeight {
 		log.L().Debug("err in BlockMetas()", zap.Error(status.Error(codes.InvalidArgument, "start height should not exceed tip height")))
-		return nil, nil
```
Comment (dustinxie) [inline api/coreservice.go:1045]: add test coverage in both `grcpserver_test.go` and `web3server_test.go`

**Review by jasonhaowa — APPROVED** [review]

**Review by maswalker — APPROVED** [review]

File: api/coreservice.go [hunk 2]
```diff
@@ -1038,11 +1038,10 @@ func (core *coreService) BlockMetas(start uint64, count uint64) ([]*iotextypes.B
 
 	var (
 		tipHeight = core.bc.TipHeight()
-		res       []*iotextypes.BlockMeta
+		res       = make([]*iotextypes.BlockMeta, 0)
 	)
 	if start > tipHeight {
-		log.L().Debug("err in BlockMetas()", zap.Error(status.Error(codes.InvalidArgument, "start height should not exceed tip height")))
-		return nil, nil
+		return nil, status.Error(codes.NotFound, "start height should not exceed tip height")
```
Comment (dustinxie) [inline api/coreservice.go:1044]: should use `codes.InvalidArgument`, same as above 2 places

File: api/web3server.go [hunk 3]
```diff
@@ -344,6 +343,10 @@ func (svr *Web3Server) getBlockByNumber(in interface{}) (interface{}, error) {
 
 	blkMetas, err := svr.coreService.BlockMetas(num, 1)
 	if err != nil {
+		st, ok := status.FromError(err)
+		if ok && st.Code() == codes.NotFound {
```
Comment (dustinxie) [inline api/web3server.go:347]: `codes.InvalidArgument`

Comment (dustinxie) [reply api/web3server.go:347]: there are also 2 other places calling `coreService.BlockMetas` in this file, do those 2 places need this fix?

Comment (Liuhaai) [reply api/web3server.go:347]: none

File: api/coreservice.go (hunk 2 above)
Comment (Liuhaai) [reply api/coreservice.go:1044]: no

**Pull request merged**